src/presenter/board-presenter.js

```diff
@@ -54,29 +53,38 @@ export default class BoardPresenter {
 
   onAddBtnCLick() {
     const newPoint = getNewPoint();
-    this.editPoint = new EditPoint(newPoint, this.waypoints.length - 1);
+    this.editPoint = new EditPoint(newPoint, this.waypoints.length - 1, true);
     render(this.editPoint, this.containerWaypoint.element, RenderPosition.AFTERBEGIN);
 
-    function onEditPointSubmit(i, update) {
-      model.addPoint(update);
-
-      this.onSortTypeChange(this.sortType);
-      this.addOnAddBtnCLick();
+    async function onEditPointSubmit(i, state) {
```

Comment:
название этой функции тоже не плохо бы поправить
```suggestion
    async function onEditPointFormSubmit(i, state) {
```

Reply:
с удалением тоже самое

Reply:
обработчик удаления так и не переименовал